orchestrator/relayer/src/batch_relaying.rs

```diff
@@ -158,10 +165,17 @@ async fn should_relay_batch(
             let cost_with_margin = get_cost_with_margin(cost, *margin);
             // we need to see how much WETH we can get for the reward token amount,
             // and compare that value to the gas cost times the margin
-            match (price, get_whitelist_amount(batch.token_contract, whitelist)) {
-                (_, Some(amount)) => amount <= batch.total_fee.amount,
+            match (price, get_whitelist_price(batch.token_contract, whitelist)) {
+                // config specifies this tokens price
+                (_, Some((whitelist_price, decimals))) => {
+                    let one = Uint256::from(1u8);
+                    let one = one.pow(decimals as u32).to_string().parse().unwrap();
```

Comment:
I don't fully get why you're doing .to_string().parse() here. Wouldn't it be better to use a conversion method/other validation instead?

Reply:
there's no conversion method, it's a hack to get around it.